EMacco/Customer-Ticketing-System
# #168825632 Change user role

#### What does this PR do?
This PR allows an admin to be able to change the role of another user

#### Description of Task to be completed?
- List all users with basic information
- Allow admin to be able to choose between user, agent, and admin

#### How should this be manually tested?
- Checkout to this branch
- Login as admin
- Click on users
- Click on the drop down against a user and select a role to change
- The user's role should have been changed

#### What are the relevant pivotal tracker stories?
[#168825632](https://www.pivotaltracker.com/story/show/168825632)

#### Screenshots (if appropriate)
![user role](https://user-images.githubusercontent.com/20377428/65857781-7327df00-e35c-11e9-8cf6-4d54a899ac7f.PNG)
